web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -219,6 +228,30 @@ class CVEAudit extends React.Component<Props, State> {
     });
   };
 
+  getPatchStatusAccuracyWarning = (row) => {
+    const dataSources: string[] = row.scanDataSources;
+    if (!dataSources) {
+      console.error("CVE audit data sources were not supplied by server.");
+      return t("Error, see console");
+    }
+
+    if (dataSources.length === 0) {
+      return t("Unknown patch status");
+    } else if (dataSources.indexOf("OVAL") === -1) {
+      return t(
+        "OVAL data was found not to be in sync, thus the server was scanned with errata metadata fetched from channels. This could mean that the operating system installed on the server is not supported by OVAL based CVE auditing or that OVAL data is not in sync, if it is the latter option please sync OVAL data as it is known to yield more accurate results."
```

Comment:
```suggestion
        "OVAL data is out of sync, so the server was scanned using errata metadata from channels. This could mean that the server's OS isn't supported by OVAL-based CVE auditing, or that OVAL data needs to be synced."
```
NitPick:This is a good  message to be added in the documentation  but for UI probably a bit too long. A bit concise message might be better.

Reply:
Thanks, I reverted back to the shorter message.